lib/rasr_cache.py

```diff
@@ -542,8 +542,8 @@ def addAttributes(self, filename, dim, duration):
         ) % (dim, duration)
         self.write_U32(self.start_recovery_tag)
         filename = "%s.attribs" % filename
-        self.write_u32(len(filename))
-        self.write_str(filename)
+        self.write_u32(len(filename.encode(self.encoding)))
```

Comment:
Two lines above, `write_U32` here `write_u32` is there a difference? Or can this be more consistent?

Reply:
OK found the functions.. `i` vs `I`???

Reply:
`int` or `unsigned int`